### PR TITLE
CI: Adding format change

### PR DESCRIPTION
**Pull request by riyadixitagra**
Due to new change in openbmc-build-scripts for the 1110 branch to clang-18, a lot of files were failing in CI format. Adding the changed format files in correspondance to CI update.

### CODE REVIEW

File: .secrets.baseline [hunk 1]
```diff
@@ -87,24 +87,6 @@
         "verified_result": null
       }
     ],
-    "oem/ibm/pfw-sms-utils/pfw_sms_menu.cpp": [
```
Comment (Pavithrab7) [inline .secrets.baseline:90]: why was this removed?

**Review by Pavithrab7 — CHANGES_REQUESTED** [review]

File: .secrets.baseline (hunk 1 above)
Comment (manojkiraneda) [reply .secrets.baseline:90]: +1

**Review by manojkiraneda — CHANGES_REQUESTED** [review]

File: .secrets.baseline (hunk 1 above)
Comment (jaypadath) [reply .secrets.baseline:90]: Rather .secrets.baseline is a generated file ? For this commit to I think that file is not needed.

Comment (riyadixitagra) [reply .secrets.baseline:90]: okk

Comment (riyadixitagra) [reply .secrets.baseline:90]: But this commit has no change other than what running the local CI caused, so I added it. Please let me know how to handle this for CI to pass?

Comment (jaypadath) [reply .secrets.baseline:90]: Riya, you can ignore it as that deleted pfw file is not yet present in 1110-ghe and it will only be checked in with my SMS changes. Secrets file was just added for CI to pass automatically. So I am good here.

**Review by jaypadath — APPROVED** [review]

File: .secrets.baseline (hunk 1 above)
Comment (manojkiraneda) [reply .secrets.baseline:90]: cool, sounds good then.

**Review by manojkiraneda — APPROVED** [review]
